src/core/store.ts

```diff
@@ -20,6 +21,7 @@ export interface RootState {
   tasks: TasksStoreSlice;
   views: ViewsStoreSlice;
   organizations: OrganizationsStoreSlice;
+  users: UsersStoreSlice;
```

Comment:
I think this should be called `user`, not `users` (and the relevant changes be made to other references to `users`). Because it will never hold other users than the current one.